src/main/java/org/opentripplanner/middleware/tripMonitor/JourneyState.java

```diff
@@ -1,13 +1,8 @@
 package org.opentripplanner.middleware.tripMonitor;
```

Comment:
Regarding a previous PR discussion about package naming, might be a good time to change the package name to all lower case.

Reply:
Renamed in 1987213b6754ffa98f4748820b628be64f9443f4